presto-docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -127,11 +127,6 @@ ownership of any schema, you can use the following rules:
 .. code-block:: json
 
     {
-      "catalogs": [
-        {
-          "allow": true
-        }
-      ],
```

Comment:
Yes this is how most of the rules today treat these today.  An empty array will always result in "no rule matched", so deny.  For no rules provided, the default is typically allow, but there are exceptions.  For every rule set the docs clearly note what happens.

In the later commits, I normalize catalog, schema, and table rules to this.  Instead of using Optionals for these in the code, I replace Optional.empty() with an allow rule, and the other is just an empty list so they get denied.